### DISCUSSION TITLE
Power loss restore

### DISCUSSION THREAD

**Discussion by kingcode94** [Koenkk/zigbee2mqtt#18042] [Devices]
Is there a way to configure zigbee2mqtt to restore all Hue lights to their previous state, including color, after a power failure? 

I've tried setting the "color_temp_startup" option, but it doesn't seem to work as expected. I have also tried all of the "power_on_behavior" options but they don't seem work.

 The colors/brightness are ignored, and the lights even turn on if they were previously off. I have this functionality working on the Hue bridge. Is there an alternative method to achieve the same result with zigbee2mqtt?

**Comment by conductorj** [comment]
@kingcode94, did you ever figure this out?  I'm interested in moving from ZHA to zigbee2mqtt and not even seeing the power on brightness setting.  I do have the option to set power_on_behavior and that seems to work but can't control power on brightness with my Hue bulbs.  This power on brightness setting is available in ZHA and works.
